pkg/configmap/configmap_test.go

```diff
@@ -30,18 +31,34 @@ func TestValidateOperatorConfigMap(t *testing.T) {
 
 func TestGetOperatorConfigMap(t *testing.T) {
 	tests := []struct {
-		name                   string
-		localObjects           []runtime.Object
-		expectedParentFolderID string
-		expectedBillingAccount string
-		expectedErr            error
-		validateResult         func(*testing.T, string, string)
-		validateErr            func(*testing.T, error, error)
+		name                     string
+		localObjects             []runtime.Object
+		expectedParentFolderID   string
+		expectedBillingAccount   string
+		expectedCCSConsoleAccess []string
+		expectedErr              error
+		validateResult           func(*testing.T, string, string)
+		validateErr              func(*testing.T, error, error)
 	}{
 		{
 			name: "Correct parentFolderID and billingAccount exist in configmap",
 			localObjects: []runtime.Object{
-				builders.NewTestConfigMapBuilder("gcp-project-operator", "gcp-project-operator", "billing123", "1234567").GetConfigMap(),
+				&corev1.ConfigMap{
+					TypeMeta: metav1.TypeMeta{
+						Kind:       "ConfigMap",
+						APIVersion: "v1",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "gcp-project-operator",
+						Namespace: "gcp-project-operator",
+					},
+					Data: map[string]string{
+						"data": `
+parentFolderID: 1234567
+billingAccount: billing123
+`,
```

Comment:
dumb question, is this formatted properly?

Reply:
Yes

Reply:
Agree it's difficult to read, maybe use a one-line json?

Reply:
Umm, why this is a problem. That's how `` is used to express yaml. It would be clear in the real yaml config.